csharp/ql/test/utils/modeleditor/PublicClass.cs

```diff
@@ -0,0 +1,26 @@
+using System;
```

Comment:
Maybe it is worth considering to introduce a couple of more tests
(1) Explicit interface implementation (see example earlier in review).
(2) Generics.